frontend/resources/translations/de-DE.json

```diff
@@ -240,15 +268,15 @@
   },
   "oAuth": {
     "err": {
-      "failedTitle": "Keine Verbindung zu Ihrem Konto möglich.",
-      "failedMsg": "Bitte prüfen, ob der Anmeldevorgang im Browser abgeschlossen wurde."
+      "failedTitle": "Keine Verbindung zum Konto möglich.",
+      "failedMsg": "Prüfe, ob der Anmeldevorgang im Browser abgeschlossen wurde."
     },
     "google": {
-      "title": "GOOGLE ANMELDUNG",
-      "instruction1": "Um Ihre Google-Kontakte zu importieren, müssen Sie diese Anwendung über Ihren Webbrowser autorisieren.",
-      "instruction2": "Kopieren Sie diesen Code in Ihre Zwischenablage, indem Sie auf das Symbol klicken oder den Text auswählen:",
-      "instruction3": "Rufen Sie den folgenden Link in Ihrem Webbrowser auf, und geben Sie den obigen Code ein:",
-      "instruction4": "Klicken Sie unten auf die Schaltfläche, wenn Sie die Anmeldung abgeschlossen haben:"
+      "title": "Google Sign-In",
+      "instruction1": "Um die Google-Kontakte zu importieren, muss die Anwendung über den Webbrowser autorisiert werden.",
+      "instruction2": "Kopiere den Code in die Zwischenablage, über das Symbol oder indem du den Text auswählst:",
```

Comment:
Kopiere den Code in die Zwischenablage. Entweder über das Symbol oder indem du den Text auswählst: